go/v1beta1/api/occurrence.go

```diff
@@ -245,8 +245,10 @@ func (g *API) DeleteOccurrence(ctx context.Context, req *gpb.DeleteOccurrenceReq
 	if err != nil {
 		return err
 	}
-	if err := g.Auth.CheckAccessAndProject(ctx, notePID, nID, NotesAttachOccurrence); err != nil {
-		return err
+	if notePID != "" && nID != "" {
```

Comment:
I'd prefer it if we checked if o.NoteName were empty instead of changing the parser.

Reply:
done